### ISSUE TITLE
PP crashes at start-up since update to 0.64

### ISSUE THREAD

**Issue by Xathros1** [flathub/info.portfolio_performance.PortfolioPerformance#66]
Update to 0.64 = since commit https://github.com/flathub/info.portfolio_performance.PortfolioPerformance/commit/cf13b53c262afa9f9a6e0cf7fd28e4bab3e6bb70

I only get a error message on start-up: 
> "An error has occurred (...) See log under ./eclipse/1412530840_linux_gtk_x86_64/configuration/xxx.log"

**Logfile (1687092293037.log)**
[1687092293037.log](https://github.com/flathub/info.portfolio_performance.PortfolioPerformance/files/11781537/1687092293037.log)
or https://paste.opensuse.org/pastes/efca65e314cf

**System info:**
OS: openSUSE Tumbleweed x86_64
Kernel: 6.3.7-1-default
DE: GNOME 44.2
WM: Mutter
Flatpak 1.14.4

**Comment by Xathros1** [comment]
found a workaround that works for me
Need to delete the ./eclipse folder:
`rm -r ~/.eclipse`
After doing that, it starts again

**Comment by Fenrihr** [comment]
Same problem. 

@Xathros1 [solution](https://github.com/flathub/info.portfolio_performance.PortfolioPerformance/issues/66#issuecomment-1596169626) works for me too.

**Log:** https://paste.opensuse.org/pastes/bb09f1965c9f

**System info:**
OS: Fedora Linux 38 (Workstation Edition)
Kernel: 6.3.5-200.fc38.x86_64
Windows Manager: Wayland
Gnome Version: 44.1
Flatpak: 1.15.4

**Comment by roshanshariff** [comment]
This is being discussed upstream at buchen/portfolio#3418.

**Comment by roshanshariff** [comment]
This should now be fixed with commit af5c0c24ed7d28648d0330c97fdc5f46351c2427.

**Comment by roshanshariff** [comment]
This issue has reappeared with Portfolio Performance version 0.65.4. For now, a workaround is to run
```
rm -r ~/.var/app/info.portfolio_performance.PortfolioPerformance/config/configuration
```

We're discussing the problem upstream at portfolio-performance/portfolio#3418.
